docs/docs/00-welcome.md

```diff
@@ -1,6 +0,0 @@
----
```

Comment:
@meynet can you add the change to /docs/_layouts/default.htm to this PR so we can close this out and merge it?